lib/atscppapi/examples/globalhook/GlobalHookPlugin.cc

```diff
@@ -19,24 +19,30 @@
 #include <iostream>
 #include <atscppapi/GlobalPlugin.h>
 #include <atscppapi/PluginInit.h>
-
+#include<../ts/Diags.h>
```

Comment:
I used it for test purpose, I wanted to write in the diags.log file. Later forgot to remove it. I will remove that